packages/suite/src/support/messages.ts

```diff
@@ -7495,57 +7495,161 @@ export default defineMessages({
         id: 'TR_LOADING_FUNDS',
         defaultMessage: 'Loading Funds...',
     },
-    TR_COINJOIN_FACT_TITLE: {
-        id: 'TR_COINJOIN_FACT_TITLE',
+    TR_LOADING_FACT_TITLE: {
+        id: 'TR_LOADING_FACT_TITLE',
         defaultMessage: 'Did you know?',
     },
-    TR_COINJOIN_FACT_ONE: {
-        id: 'TR_COINJOIN_FACT_ONE',
+    TR_LOADING_FACT_0: {
+        id: 'TR_LOADING_FACT_0',
+        description: '140 symbols max',
```

Comment:
Does it need to be specified in each fact? 😆

Reply:
Better be safe, I thought 😅